python/tbb/__main__.py

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env python3
-#
-# Copyright (c) 2016-2021 Intel Corporation
+# Copyright (c) 2016-2022 Intel Corporation
```

Comment:
```suggestion
# Copyright (c) 2016-2023 Intel Corporation
```

Reply:
Yeah, sorry for the typo. I've force-pushed a fix.